plugins/user/joomla/joomla.php

```diff
@@ -426,6 +426,15 @@ public function onUserLogout($user, $options = [])
 
 			/** @var SessionManager $sessionManager */
 			$sessionManager = Factory::getContainer()->get('session.manager');
+			
+			// Handle resource
```

Comment:
```suggestion

			// Handle resource
```
Make PHPCS in Drone happy.